### ISSUE TITLE
UI Search Results for #hastagged posts

### ISSUE THREAD

**Issue by jterps08** [PowerlineApp/powerline-mobile#108]
The format of the search results is the same as a very old version of the newsfeed. Ideally search results (for hashtagged posts) should match the same UI design as the current newsfeed. very low priority.

**Comment by petervojtek** [comment]
@jterps08 , this is how the search results for #hashtagged posts looks like now:

<img width="770" alt="screen shot 2016-07-22 at 11 29 03" src="https://cloud.githubusercontent.com/assets/225506/17052871/9440a012-4fff-11e6-9f6f-8f00719fee05.png">

For leader-generated content this issue is blocked by #130 -- i.e. I am unable to create leader-content and thus unable to test how it is rendered.

For non-leader content (i.e. micro-petitions), this search screen and associates API endpoints seems to be completely unware of micro-petitions (the pokemon post was created as User Post, i.e. `micro-petition:quorum`), however it is provided by `GET https://api-dev.powerli.ne/api/search/by-hash-tags?query=%23pokemons`  as `petition`:

```
{"petitions":[{"quorum_count":1,"id":220,"title":"","petition_body":"give us more #pokemons","group":{"id":285,"group_type":3,"avatar_file_path":"https://api-dev.powerli.ne/bundles/civixfront/img/default_group.png","official_title":"San Diego"},"group_id":285,"created_at":"Fri, 22 Jul 2016 09:04:13 +0000","expire_at":"Sat, 23 Jul 2016 09:04:12 +0000","user":{"full_name":"Peter10 Peter10","id":73,"username":"Peter10","first_name":"Peter10","last_name":"Peter10","avatar_file_name":"http://devpowerline.s3-website-eu-west-1.amazonaws.com/avatars/577cb68e6f276.jpeg"},"publish_status":0,"type":"quorum","cached_hash_tags":["#pokemons"],"responses_count":0,"created_at_date":"2016-07-22T09:04:13.000Z","expired_at_date":"2016-07-23T09:04:12.000Z","count_to_quorum":1,"completed_percent":0,"owner":{"id":73,"avatar":"http://devpowerline.s3-website-eu-west-1.amazonaws.com/avatars/577cb68e6f276.jpeg","type":"user","name":"Peter10 Peter10"}}]}
```

So for the non-leader content, first the backend should be altered and then we can adjust the frontend. I assigned this issue to you Jesse to make the decision whether backend team should pay attention to this issue right now, or rather later..

**Comment by jterps08** [comment]
Thanks for the analysis, Peter... I think we are going to have to delay working on this issue for now. If Igor is able to finish other more important items, then we can try to  alter the backend so that the frontend can be updated.

I have asked Austin to create test data for leader-generated content.

**Comment by petervojtek** [comment]
duplicate of #215
